src/platform/packages/shared/response-ops/rule_form/src/rule_definition/rule_investigation_guide_editor.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIconTip,
+  EuiMarkdownEditor,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+interface Props {
+  setRuleParams: (v: { investigation_guide: { blob: string } }) => void;
+  value: string;
+}
+
+export function InvestigationGuideEditor({ setRuleParams, value }: Props) {
+  return (
+    <>
+      <EuiFlexGroup gutterSize="xs" alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiText>
+            <strong>
+              <FormattedMessage
+                id="responseOpsRuleForm.investigationGuide.editor.title"
+                defaultMessage="Investigation Guide"
+              />
+            </strong>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiIconTip content="Include a guide or useful information for addressing alerts created by this rule" />
```

Comment:
this should be translated

Reply:
[7436061](https://github.com/elastic/kibana/pull/217106/commits/743606149f6bfb69e65eab3b067b09bd673f866b)